commcare_connect/templates/tailwind/pages/dashboard.html

```diff
@@ -0,0 +1,31 @@
+<!-- dashbaord.html -->
+{% extends 'tailwind/base.html' %}
+
+{% block content %}
+
+<div id="content-area" class="md:max-w-screen-2xl mx-auto flex flex-col lg:flex-row gap-4 min-h-screen"> 
```

Comment:
I don't understand why you are duplicating the entire dashboard from scratch again. We have already integrated this page and we have also been iterating that page already. https://github.com/dimagi/commcare-connect/pull/537/files#diff-4ebc163b53db32df54115682ddd4feeafdf599a51d076aacc635dc73db9b7e16

This undoes all the work we did on dashboard from our end.